pustaka/variables.tex

```diff
@@ -0,0 +1,47 @@
+% Atur variabel berikut sesuai namanya
+
+% nama
+\newcommand{\name}{Elon Reeve Musk}
+\newcommand{\authorname}{Musk, Elon Reeve}
+\newcommand{\nickname}{Elon}
+\newcommand{\advisor}{Nikola Tesla, S.T., M.T}
+\newcommand{\coadvisor}{Wernher von Braun, S.T., M.T}
+\newcommand{\examinerone}{Dr. Galileo Galilei, S.T., M.Sc}
+\newcommand{\examinertwo}{Friedrich Nietzsche, S.T., M.Sc}
+\newcommand{\examinerthree}{Alan Turing, ST., MT}
+\newcommand{\headofdepartment}{Dr. Supeno Mardi Susiki Nugroho, S.T., M.T}
```

Comment:
I suggest to use fictional name for this person :)
(and the NIP too)

Reply:
dumbledore lesgoo..

Reply:
oke dumbledore